server/src/server.py

```diff
@@ -22,7 +22,7 @@ def default(self, *args, **kwargs):
 
     @cherrypy.expose
     def index(self):
-        page = open(SITE_ROOT + "/test/index.html", "r")
+        page = open(SITE_ROOT + "/public/index.html", "r")
```

Comment:
Not sure how CherryPi -> React will actually work but we can figure this out once we start working on the API